ambry-cloud/src/main/java/com/github/ambry/cloud/CloudContainerDeletionSyncTask.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2020 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.cloud;
+
+import com.github.ambry.account.AccountService;
+import com.github.ambry.account.Container;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Set;
+import org.apache.helix.task.Task;
+import org.apache.helix.task.TaskResult;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Helix task to get the deleted containers information from {@link com.github.ambry.account.AccountService} and update
+ * it in CosmosDb DeleteContainers table.
+ */
+public class CloudContainerDeletionSyncTask implements Task {
```

Comment:
This task, like the updateDeletedContainers() methods, seems fairly specific to the Azure/Cosmos impl.  Can we move this to the cloud/azure package (make it Azure only), remove the method from CloudDestination interface, and call the method directly on the AzureContainerCompactor?